includes/class-admin.php

```diff
@@ -314,7 +318,11 @@ public static function save_user_description( $user_id ) {
 	public static function enqueue_scripts( $hook_suffix ) {
 		if ( false !== strpos( $hook_suffix, 'activitypub' ) ) {
 			wp_enqueue_style( 'activitypub-admin-styles', plugins_url( 'assets/css/activitypub-admin.css', ACTIVITYPUB_PLUGIN_FILE ), array(), '1.0.0' );
-			wp_enqueue_script( 'activitypub-admin-styles', plugins_url( 'assets/js/activitypub-admin.js', ACTIVITYPUB_PLUGIN_FILE ), array( 'jquery' ), '1.0.0', false );
+			wp_enqueue_script( 'activitypub-admin-script', plugins_url( 'assets/js/activitypub-admin.js', ACTIVITYPUB_PLUGIN_FILE ), array( 'jquery' ), '1.0.0', false );
+		}
+
+		if ( 'index.php' === $hook_suffix ) {
+			wp_enqueue_style( 'activitypub-admin-styles', plugins_url( 'assets/css/activitypub-admin.css', ACTIVITYPUB_PLUGIN_FILE ), array(), '1.0.0' );
```

Comment:
You need to bust the cache, for me it first looked like this:

![Screenshot 2024-06-04 at 10 14 04](https://github.com/Automattic/wordpress-activitypub/assets/203408/f1db1f60-f931-4630-8ec8-0d3806c55c9c)

```suggestion
			wp_enqueue_style( 'activitypub-admin-styles', plugins_url( 'assets/css/activitypub-admin.css', ACTIVITYPUB_PLUGIN_FILE ), array(), get_plugin_version() );
```

Reply:
Maybe you should add caching to `get_plugin_version()` or `get_plugin_meta()` if it gets called more often.